jkq/qfr/qiskit/QasmQobjExperiment.hpp

```diff
@@ -0,0 +1,178 @@
+/*
+ * This file is part of JKQ QFR library which is released under the MIT license.
+ * See file README.md or go to http://iic.jku.at/eda/research/quantum/ for more information.
+ */
+
+#ifndef QFR_QASMQOBJEXPERIMENT_HPP
+#define QFR_QASMQOBJEXPERIMENT_HPP
+
+namespace qiskit {
+    class QasmQobjExperiment {
+    public:
+        static void import(qc::QuantumComputation& qc, const py::object& circ) {
+            qc.reset();
+
+            py::object pyQasmQobjExperiment = py::module::import("qiskit.qobj").attr("QasmQobjExperiment");
+
+            if (!py::isinstance(circ, pyQasmQobjExperiment)) {
+                throw qc::QFRException("[import] Python object needs to be a Qiskit QasmQobjExperiment");
+            }
+
+            auto&& header = circ.attr("header");
+            //            auto&& config       = circ.attr("config");
+            auto&& instructions = circ.attr("instructions");
+
+            auto&& circQregs = header.attr("qreg_sizes");
+            for (const auto& qreg: circQregs) {
+                qc.addQubitRegister(qreg.cast<py::list>()[1].cast<dd::QubitCount>(), qreg.cast<py::list>()[0].cast<std::string>().c_str());
```

Comment:
`py::list` has no range-checked `at` method. But unless Qiskit changes its format again, these entries are guaranteed to exist. In case Qiskit changes, we have to adapt the bindings anyway.

Reply:
okay